regolith/schemas.py

```diff
@@ -877,10 +922,60 @@
             "type": "integer",
         },
     },
+    "contacts": {
+        "_description": {"a lighter version of people.  Fewer required fields"
+                         "for capturing people who are less tightly coupled"
+                         },
+        "_id": {
+            "description": "id of the person, e.g., first letter first name "
+                           "plus last name, but unique",
+            "required": True,
+        },
+        "aka": {
+            "required": False,
+            "type": "list",
+            "description": "other names for the person",
+        },
+        "department": {
+            "description": "Department at the institution",
+            "type": "string",
+            "required": False,
+        },
+        "email": {
+            "description": "Contact email for the contact",
+            "type": "string",
+            "required": False,
+        },
+        "institution": {
+            "description": "the institution where they are located.  This is"
+                           "required for building a COI list of coauthors, but"
+                           "not in general.  It can be institute id or anything"
+                           "in the aka or name",
+            "required": False,
+            "type": "string"
+        },
+        "name": {
+            "description": "the person canonical name",
+            "required": True,
+            "type": "string",
+        },
+        "notes": {
+            "description": "notes about the person",
+            "required": False,
+            "anyof_type": ["list", "string"]
+        },
+        "title": {
+            "description": "how the person is addressed",
+            "required": False,
+            "type": "string",
+        },
+        "required": False,
+        "type": "dict",
```

Comment:
Are these two lines needed?

Reply:
I removed....let's see if that fixes it.